tests/integration/questionnaire/test_questionnaire_list_collector.py

```diff
@@ -231,3 +231,26 @@ def test_cancel_text_displayed_on_edit_block_if_exists(self):
         self.get(change_link)
 
         self.assertInBody("Don’t need to change anything?")
+
+    def test_summary_does_not_display_answers(self):
```

Comment:
The name of this test isn't clear. Something like `test_questionnaire_summary_does_not_display_custom_section_summary` would be better (even though it's a bit wordy).